iFixit/react-commerce
# Bug: Fix repeated facets

Previously, we  had some repeated facets being displayed. Although I don't know if this is the solution we are looking for here but this PR basically makes sure that we do not display any facets that have an `item.count` of 0.

## QA:
- Visit the preview and add `/Parts/iMac_Intel_17"/Hard_Drives_` 
- Make sure there are no repeated facets as mentioned in the issue.

Closes #698 

CC: @erinemay @sterlinghirsh
